vmcnet/train/runners.py

```diff
@@ -604,6 +604,12 @@ def run_molecule() -> None:
         )
 
 
+
+def run_hubbard() -> None:
```

Comment:
Probably good for now just to leave this out so that the PR is a bit cleaner (only adds to the python API) and re-introduce this runner in a future PR.